caffe2/python/gradient_check_test.py

```diff
@@ -269,6 +269,28 @@ def testCos(self):
                 res, grad, grad_estimated = checker.CheckSimple(op, [X], 0, [0])
                 self.assertTrue(res)
 
+
+class TestAcos(test_util.TestCase):
```

Comment:
don't add new tests here, use folder caffe2/python/operator_test, create new files for your operators (or group of operators). Then use helpers such as assertReferenceChecks and assertGradientChecks. Fbgs those for more examples